test/extended/util/db/mongodb.go

```diff
@@ -0,0 +1,62 @@
+package db
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/openshift/origin/test/extended/util"
+)
+
+// MongoDB is a MongoDB helper for executing commands.
+type MongoDB struct {
+	PodName       string
+	MasterPodName string
+}
+
+// NewMongoDB queries OpenShift for a pod with given name, saving environment
+// variables like username and password for easier use.
+func NewMongoDB(podName, masterPodName string) util.Database {
+	if masterPodName == "" {
+		masterPodName = podName
+	}
+	return &MongoDB{
+		PodName:       podName,
+		MasterPodName: masterPodName,
+	}
+}
+
+func (m MongoDB) GetPodName() string {
+	return m.PodName
```

Comment:
@php-coder I don't see a use-case where you will want to return something else than m.PodName, if there is such use-case then i'm fine keeping it